filestore/protoboards_test.go

```diff
@@ -0,0 +1,183 @@
+package filestore_test
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/filestore"
+	clog "github.com/influxdata/chronograf/log"
+)
+
+func Test_Protoboard_All(t *testing.T) {
+	t.Parallel()
+	var tests = []struct {
+		Existing []chronograf.Protoboard
+		Err      error
+	}{
+		{
+			Existing: []chronograf.Protoboard{
+				{ID: "1"},
+				{ID: "2"},
+			},
+			Err: nil,
+		},
+		{
+			Existing: []chronograf.Protoboard{},
+			Err:      nil,
+		},
+		{
+			Existing: nil,
+			Err:      errors.New("Error"),
+		},
+	}
+	for i, test := range tests {
+		fp, _ := MockProtoboards(test.Existing, test.Err)
+		protoboards, err := fp.All(context.Background())
+		if err != test.Err {
+			t.Errorf("Test %d: protoboards all error expected: %v; actual: %v", i, test.Err, err)
+		}
+		if !reflect.DeepEqual(protoboards, test.Existing) {
+			t.Errorf("Test %d: Protoboards should be equal; expected %v; actual %v", i, test.Existing, protoboards)
+		}
+	}
+}
+
+func Test_Protoboard_Get(t *testing.T) {
```

Comment:
The `_`s in this test name are a little weird. Was there a particular pattern you were following?

Reply:
copied from layouts code